practical-07/src/route/route.js

```diff
@@ -0,0 +1,31 @@
+import { createRouter, createWebHistory } from 'vue-router'
+const router = createRouter({
+    history: createWebHistory(),
+    routes: [{
+            path: '/',
+            name: 'HomeView',
```

Comment:
Codacy has a fix for the issue: [Replace `············name:·'HomeView'` with `······name:·"HomeView"`](https://app.codacy.com/gh/mansinakrani/vuejs-practicals/pullRequest?prid=9375994)
```suggestion
      name: "HomeView",
```